modules/apps/frontend-editor/frontend-editor-ckeditor-sample-web/src/main/resources/META-INF/resources/js/CKEditor5ReactClassicEditor.tsx

```diff
@@ -5,14 +5,14 @@
 
 import {EditorConfig} from 'ckeditor5';
 import {
-	AdvancedPreset,
 	CKEditor5ClassicEditor as ClassicEditor,
+	advancedClassicEditorConfig,
 } from 'frontend-editor-ckeditor-web';
 import React from 'react';
 
 const CKEditor5ReactClassicEditor = () => {
 	const config: EditorConfig = {
-		...AdvancedPreset,
+		...advancedClassicEditorConfig,
```

Comment:
I renamed this in two ways:
- camelCase, as it is an object, not a class or a component
- `*EditorConfig`, to fit `EditorConfig` type. A bit of borrowing from Java convention. I think this will be a more natural to devs then arbitrary `Preset`.

Reply:
what makes a preset different from an arbirary editor config, other than it's provided ootb? I mean, is it important to keep "Preset" as part of the imprted symbol?

Reply:
No difference. TS even enforces it, as you must return [an object of EditorConfig type](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/js/ckeditor5/presets/AdvancedPreset.ts#L9). The naming here is arbitrary, it doesn't _have_ to be `*Preset` or `*EditorConfig`. It's just what we think makes more sense. In my opinion, `*Preset` does make more sense from the product side. But to the developers, who will be using the API, `*EditorConfig` is far more informative of what is the content of the import.